<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_loadcitra">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>load citra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>411</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_prosescitra">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>520</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>proses citra</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperasi_titik">
    <property name="title">
     <string>Operasi Titik</string>
    </property>
    <addaction name="actionOperasi_Pencerahan"/>
    <addaction name="actionSimple_Contrast"/>
    <addaction name="actionContrast_Streching"/>
    <addaction name="actionnegative"/>
    <addaction name="actionoperasi_biner"/>
   </widget>
   <addaction name="menuOperasi_titik"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOperasi_Pencerahan">
   <property name="text">
    <string>Operasi Pencerahan</string>
   </property>
  </action>
  <action name="actionSimple_Contrast">
   <property name="text">
    <string>Simple Contrast</string>
   </property>
  </action>
  <action name="actionContrast_Streching">
   <property name="text">
    <string>Contrast Streching</string>
   </property>
  </action>
  <action name="actionNegative_Triggered">
   <property name="text">
    <string>Negative Triggered</string>
   </property>
  </action>
  <action name="actionnegative">
   <property name="text">
    <string>negative</string>
   </property>
  </action>
  <action name="actionoperasi_biner">
   <property name="text">
    <string>operasi biner</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
